quickwit/quickwit-control-plane/src/control_plane.rs

```diff
@@ -1681,10 +1681,12 @@ mod tests {
             .unwrap();
         assert!(indexing_tasks.is_empty());
 
-        let results: Vec<ApplyIndexingPlanRequest> =
-            client_inbox.drain_for_test_typed::<ApplyIndexingPlanRequest>();
-        assert_eq!(results.len(), 1);
-        assert!(results[0].indexing_tasks.is_empty());
+        let apply_plan_requests = client_inbox.drain_for_test_typed::<ApplyIndexingPlanRequest>();
+        assert!(!apply_plan_requests.is_empty());
```

Comment:
was the test too strict in some instances?

Reply:
Yes, sometimes the actor has time to spin more than once, and two messages are emitted, which causes the test to be flaky.